src/flow/maximum_flow.jl

```diff
@@ -1,3 +1,4 @@
+import LightGraphs.SimpleGraphs.SimpleDiGraph
```

Comment:
I changed them all to `DiGraph`. The danger here is if `DiGraph` becomes something else (say, with metadata) then this function can get very expensive.